test/extended/prometheus/prometheus.go

```diff
@@ -213,7 +213,7 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
 			tests := map[string][]metricTest{
 				// should be checking there is no more than 1 alerts firing.
 				// Checking for specific alert is done in "should have a Watchdog alert in firing state".
-				`sum(ALERTS{alertstate="firing"})`: {metricTest{greaterThanEqual: false, value: 2}},
+				`ALERTS{alertstate="firing"}`: {metricTest{greaterThanEqual: false, value: 0}},
```

Comment:
In a healthy cluster we always have a Watchdog alert firing, so this should check for `1`, not `0`. As it is described in comment above :)

Reply:
Oh, you are checking things, nvm my comment.